src/game/QuizBuilder.ts

```diff
@@ -43,11 +45,11 @@ export class QuizBuilder {
         questions.some(questions => {
             let checkRepeatedQuestions: string[] = [];
             questions.forEach(question => {
-                if (!correctAnsweredQuestions.includes(question.question)){
+                if (!correctAnsweredQuestions.includes(question.question)) {
                     checkRepeatedQuestions.push(question.question)
-                    }
-                })
-            if (checkRepeatedQuestions.length < QuizBuilder.QUESTIONS_PER_TYPE){
+                }
+            })
+            if (checkRepeatedQuestions.length < QuizBuilder.QUESTIONS_PER_TYPE) {
```

Comment:
quizable = checkRepeatedQuestions.length < QuizBuilder.QUESTIONS_PER_TYPE;

What do you think?

Reply:
after changing the tests failed